packages/api/src/brevo-contact/brevo-contact-import.service.ts

```diff
@@ -11,13 +12,29 @@ import { TargetGroupInterface } from "../target-group/entity/target-group-entity
 import { TargetGroupsService } from "../target-group/target-groups.service";
 import { EmailCampaignScopeInterface } from "../types";
 
-class ValidateableRow {
+class BasicValidateableRow {
     @IsEmail()
     @IsNotEmpty()
     email: string;
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    [key: string]: any;
+    [key: string]: string;
+}
+
+export interface CSVImportInformation {
+    created: number;
+    updated: number;
+    failed: number;
+    failedColumns: unknown[];
+    errorMessage?: string;
+}
+
+interface ImportContactsFromScvParams {
```

Comment:
```
interface ImportContactsFromCsvParams {
```

Reply:
done in: 7254d7fd4219199315a3b0834f01a6a6537c6304